HISTORY.md

```diff
@@ -3,7 +3,8 @@
 
   * Require new HISTORY.md entry for each PR.
     ([#171](https://github.com/mlpack/ensmallen/pull/171),
-     [#172](https://github.com/mlpack/ensmallen/pull/172)).
+     [#172](https://github.com/mlpack/ensmallen/pull/172),
+     [#175](https://github.com/mlpack/ensmallen/pull/175)).
```

Comment:
Pretty sure, this is the last time we have to add another PR number to the list.

Reply:
Famous last words :)